ci/install_dependencies.sh

```diff
@@ -24,23 +24,24 @@ distro_id=$(grep '^ID=' /etc/os-release|awk -F = '{print $2}'|sed 's/\"//g')
 
 case "$distro_id" in
     'fedora')
-        dnf -y --refresh install gcc-c++ clang cmake make ccache expat-devel zlib-devel libssh-devel libcurl-devel gtest-devel which dos2unix
+        dnf -y --refresh install gcc-c++ clang cmake make ccache expat-devel zlib-devel libssh-devel libcurl-devel gtest-devel gmock-devel which dos2unix
         ;;
 
     'debian')
+        echo "deb http://deb.debian.org/debian stretch-backports main contrib non-free" >> /etc/apt/sources.list
         apt-get update
-        apt-get install -y cmake g++ clang make ccache python3 libexpat1-dev zlib1g-dev libssh-dev libcurl4-openssl-dev libgtest-dev libxml2-utils
+        apt-get install -y cmake g++ clang make ccache python3 libexpat1-dev zlib1g-dev libssh-dev libcurl4-openssl-dev libgtest-dev libgmock-dev libxml2-utils
```

Comment:
```suggestion
        apt-get install -y cmake g++ clang make ccache python3 libexpat1-dev zlib1g-dev libssh-dev libcurl4-openssl-dev googletest libxml2-utils
```
The new package name is just: googletest
```
root@7fbc3ab1d982:/# cat /etc/issue 
Debian GNU/Linux 9 \n \l

root@7fbc3ab1d982:/# apt-file search gmock/gmock.h
googletest: /usr/include/gmock/gmock.h
googletest: /usr/src/googletest/googlemock/include/gmock/gmock.h

root@7fbc3ab1d982:/# apt-file search gtest/gtest.h
googletest: /usr/include/gtest/gtest.h
googletest: /usr/src/googletest/googletest/include/gtest/gtest.h
```